README.md

```diff
@@ -20,16 +20,23 @@ customPlaybooks:
 # Change the following line according to your needs
 - triggers:
   - on_prometheus_alert:
-      alert_name: TestAlert
+      alert_name: KubeJobFailed
```

Comment:
Can we use an alert that is *not* job-related. It's a little confusing that we're running a Job as a result but also triggering this on a failed job. Best to change the trigger and keep everything clear with one use of jobs here (the real one)